frontend/packages/console-dynamic-plugin-sdk/src/extensions/console-types.ts

```diff
@@ -290,6 +290,7 @@ export type NavPage = {
 export type HorizontalNavProps = {
   resource?: K8sResourceCommon;
   pages: NavPage[];
+  customData?: any;
```

Comment:
Is the type really `any`? Not even `object`?

I'd have to look at the code to see how it's used, but presumably this is passed as props to the underlying component?

Reply:
And exactly, within the `HorizontalNav` component its being passed into a child component of an `ErrorBoundaryPage` during the construction of routes, see [here]. (https://github.com/openshift/console/blob/f6a592d8adfdff417a3c1f2e3b29d12b52f6190a/frontend/public/components/utils/horizontal-nav.tsx#L342)